src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <RootNamespace>Microsoft.Extensions.AI</RootNamespace>
-    <Description>Abstractions for generative AI.</Description>
+    <Description>Abstractions representing generative AI components.</Description>
     <Workstream>AI</Workstream>
   </PropertyGroup>
 
   <PropertyGroup>
-    <Stage>preview</Stage>
+    <Stage>normal</Stage>
```

Comment:
Never seen this property before. Is it dotnet/extensions specific?

Reply:
Yes